lib/Widgets/MessageDialog.vala

```diff
@@ -253,4 +263,45 @@ public class Granite.MessageDialog : Gtk.Dialog {
         action_area.margin = 6;
         action_area.margin_top = 14;
     }
+
+    /**
+     * Shows a terminal-like widget for error details that can be expanded by the user.
+     * 
+     * This method can be useful to provide the user extended error details in a
+     * terminal-like text view. Note that this method adds the widget itself
+     * to {@link Granite.MessageDialog.custom_bin}. Calling this method with a manually added widget
+     * to {@link Granite.MessageDialog.custom_bin} will result in a critical message and no
+     * addition.
+     * 
+     * Subsequent calls to this method will change the error message to a new one.
+     * 
+     * @param error_message the detailed error message to display
+     */
+    public void show_error_expander (string error_message) {
```

Comment:
I think this should be named something that doesn't imply the UI in case we want to change the way errors are presented in the future